lib/ably/rest/client.rb

```diff
@@ -434,6 +439,10 @@ def send_request(method, path, params, options)
         max_retry_duration = http_defaults.fetch(:max_retry_duration)
         requested_at       = Time.now
         retry_count        = 0
+        if @add_request_ids
```

Comment:
It's bad practice to read an instance variable like this when it's available as a reader i.e. `if @add_request_idds` would always be false due to a type, whereas `if add_request_idds` would fail.